### PR TITLE
DEVDOCS-5794 [update]: Catalog API, add terms of service and policy

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5794]

## What changed?
Update the following endpoints to include terms of service and privacy policy:
- [Create a Product](https://developer.bigcommerce.com/docs/rest-catalog/products#create-a-product) 
- [Update Products Batch](https://developer.bigcommerce.com/docs/rest-catalog/products#update-products-batch)
- [Update a Product](https://developer.bigcommerce.com/docs/rest-catalog/products#update-a-product)
- [Create a Product Video](https://developer.bigcommerce.com/docs/rest-catalog/products/videos#create-a-product-video)
- [Update a Product Video](https://developer.bigcommerce.com/docs/rest-catalog/products/videos#update-a-product-video)

[DEVDOCS-5794]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5794?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by DmytroSirenko — DISMISSED** [review]

File: reference/catalog/products_catalog.v3.yml [hunk 1]
```diff
@@ -2732,7 +2732,9 @@ paths:
         * id
 
         Publicly accessible URLs are valid parameters.
-        Videos must be loaded through YouTube at this time.
+        
+        The Catalog API integrates with third-party Youtube, and videos must be loaded through YouTube at this time.
+        The [Youtube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```
Comment (bc-traciporter) [inline reference/catalog/products_catalog.v3.yml:2737]:
```suggestion
        The [YouTube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```

File: reference/catalog/products_catalog.v3.yml [hunk 2]
```diff
@@ -2950,6 +2952,9 @@ paths:
 
         **Read-Only Fields**
         * id
+
+        The Catalog API integrates with third-party Youtube, and videos must be loaded through YouTube at this time.
```
Comment (bc-traciporter) [inline reference/catalog/products_catalog.v3.yml:2956]:
```suggestion
        The Catalog API integrates with third-party YouTube, and you must load videos through YouTube now.
```

File: reference/catalog/products_catalog.v3.yml [hunk 3]
```diff
@@ -2950,6 +2952,9 @@ paths:
...
+        The [Youtube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```
Comment (bc-traciporter) [inline reference/catalog/products_catalog.v3.yml:2957]:
```suggestion
        The [YouTube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```

File: reference/catalog/products_catalog.v3.yml [hunk 4]
```diff
@@ -8095,6 +8100,9 @@ components:
             $ref: '#/components/schemas/productImage_Full'
         videos:
           type: array
+          description: |
+            The Catalog API integrates with third-party Youtube.
```
Comment (bc-traciporter) [inline reference/catalog/products_catalog.v3.yml:8104]:
```suggestion
            The Catalog API integrates with third-party YouTube.
```

File: reference/catalog/products_catalog.v3.yml [hunk 5]
```diff
@@ -8095,6 +8100,9 @@ components:
...
+            The [Youtube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```
Comment (bc-traciporter) [inline reference/catalog/products_catalog.v3.yml:8105]:
```suggestion
            The [YouTube Terms of Service](https://www.youtube.com/t/terms) and [Google Privacy Policy](https://policies.google.com/privacy) apply, as indicated in our [Privacy Policy](https://www.bigcommerce.com/privacy/) and [Terms of Service](https://www.bigcommerce.com/terms/).
```

**Review by bc-traciporter — COMMENTED** [review]
I left suggestions.

File: reference/catalog/products_catalog.v3.yml [hunk 6]
```diff
@@ -2732,7 +2732,9 @@ paths:
         * id
 
         Publicly accessible URLs are valid parameters.
-        Videos must be loaded through YouTube at this time.
+        
+        The Catalog API integrates with third-party Youtube, and videos must be loaded through YouTube at this time.
```
[the remark this answers is not in the document]
Comment (bc-andreadao) [reply reference/catalog/products_catalog.v3.yml]: thanks, I used your suggestion, but reworded it to:
`The Catalog API integrates with third-party YouTube, and you must load videos through YouTube.`

File: reference/catalog/products_catalog.v3.yml (hunk 2 above)
Comment (bc-andreadao) [reply reference/catalog/products_catalog.v3.yml]: same here, I reworded your suggestion to: 

`The Catalog API integrates with third-party YouTube, and you must load videos through YouTube.`

**Review by bc-traciporter — APPROVED** [review]
LGTM

**Pull request merged**